source/endpoints/groups.md.erb

```diff
@@ -116,7 +116,7 @@ A group is a top-level container for your URLs and global views.
 ## List groups
 
 ```shell
-  curl 'https://api.nightwatch.io/api/v1/url_groups?access_token=ACCESS_TOKEN' \
+  curl 'https://api.nightwatch.io/api/v1/url_groups?access_token=ACCESS_TOKEN' \\
```

Comment:
Why double backslashes?

Reply:
When using only one backslash, the rendition gets broken, like this:

![image](https://user-images.githubusercontent.com/273660/59148073-1185e280-8a04-11e9-96cc-8ab52a96a562.png)

But I discovered in some cases it also works with a single backslash. With double backslashes it works alway. So I opted into using double backslashes everywhere.